docs/relational-databases/system-dynamic-management-views/sys-dm-hadr-instance-node-map-transact-sql.md

```diff
@@ -15,7 +15,7 @@ dev_langs:
   - "TSQL"
 helpviewer_keywords: 
   - "Availability Groups [SQL Server], monitoring"
-  - "Availability Groups [SQL Server], WSFC clusters"
+  - "Availability Groups [SQL Server], WSFC"
```

Comment:
Just need to verify if and how the change to this metadata field might change some functionality.